src/Text/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -5,39 +5,40 @@ exports[`Text should match snapshot diff 1`] = `
 - First value
 + Second value
 
-@@ -3,24 +3,28 @@
+@@ -3,17 +3,29 @@
       Array [
         Object {
           \\"fontFamily\\": \\"Roboto\\",
           \\"margin\\": 0,
         },
--       undefined,
--       undefined,
-        Object {
--         \\"fontSize\\": 14,
++       Object {
 +         \\"fontStyle\\": \\"italic\\",
-        },
-        Object {
--         \\"fontSize\\": 14,
++       },
++       Object {
 +         \\"textTransform\\": \\"uppercase\\",
-        },
-        Object {
--         \\"color\\": \\"#46515e\\",
++       },
++       Object {
 +         \\"fontWeight\\": \\"700\\",
-        },
-        Object {
--         \\"textAlign\\": \\"left\\",
++       },
++       Object {
 +         \\"fontSize\\": 16,
-        },
++       },
 +       Object {
 +         \\"color\\": \\"#7f91a8\\",
 +       },
 +       Object {
 +         \\"textAlign\\": \\"center\\",
 +       },
         undefined,
+-       undefined,
+-       undefined,
+-       undefined,
+-       undefined,
+-       undefined,
+-       undefined,
```

Comment:
probably will be nice have there some values instead of `undefined`

Reply:
Okay, I will improve the implementation to avoid these `undefined` from popping up.